src/EnergyPlus/EMSManager.cc

```diff
@@ -271,42 +259,19 @@ namespace EMSManager {
         //                      added calling point argument and logic.
         //                      Collapsed SimulateEMS into this routine
 
-        // PURPOSE OF THIS SUBROUTINE:
-
-        // METHODOLOGY EMPLOYED:
-        // Standard EnergyPlus methodology.
-
-        // Using/Aliasing
-        using OutputProcessor::MeterType;
-        using OutputProcessor::RealVariables;
-        using OutputProcessor::RealVariableType;
-        using RuntimeLanguageProcessor::BeginEnvrnInitializeRuntimeLanguage;
-        using RuntimeLanguageProcessor::EvaluateStack;
-
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-
-        int ErlVariableNum;    // local index
-        int ProgramManagerNum; // local index and loop
-        int ErlProgramNum;     // local index
-        int ActuatorUsedLoop;  // local loop
-        int EMSActuatorVariableNum;
-
-        int tmpInteger;
-        //  INTEGER  :: ProgramNum
-
         anyProgramRan = false;
         if (!state.dataGlobal->AnyEnergyManagementSystemInModel) return; // quick return if nothing to do
 
         if (iCalledFrom == EMSCallFrom::BeginNewEnvironment) {
-            BeginEnvrnInitializeRuntimeLanguage(state);
+            RuntimeLanguageProcessor::BeginEnvrnInitializeRuntimeLanguage(state);
```

Comment:
How about we combine the namespaces `DataRuntimeLanguage` and `RuntimeLanguageProcessor` into a single namespace called `ERL`? Or combine those with `EMSManager` into a single namespace named `EMS`?